Clinical-Genomics/scout
# Remove and/or restore removed gene panels from GUI

This PR adds the ability for gene panel maintainers and admins to remove gene panels from the gene panels view in Scout GUI. Removed panels are marked as "hidden" in the database which prevents them from being associated with new samples. If a sample is associated with a removed panel, the panel will be displayed in the case view as removed. The user will still be able to inspect the content of a removed panel but will not be able to use it as a filter.

Admins can restore removed panels from the panel view.

**How to test**:
1. Setup demo environment
2. Create two user, one admin and one normal user
3. Login as admin
4. Ensure that Test panel has been added are associated with case 643594
5. Remove test panel by clicking on "trash" icon and ensure that trash icon has changed to restore icon
6. Goto case 643594 and ensure that Test pannel is displayed as removed and is not showing up in the filters.
7. Log in as a normal user and goto /panels. Ensure that Test panel does not show up in the list.
8. Login as admin and restore Test panel.
9. Ensure that the panel functions normally for case 643594 

**Review:**
